src/components/v5/common/CompletedAction/partials/MeatballMenu/MeatballMenu.tsx

```diff
@@ -23,6 +25,48 @@ const MeatballMenu: FC<MeatballMenuProps> = ({
     ],
   } = useActionSidebarContext();
 
+  const items = useMemo(() => {
+    const menuItems: MeatBallMenuItem[] = [];
+
+    if (showRedoItem) {
+      menuItems.push({
+        key: '1',
+        label: formatText({ id: 'completedAction.redoAction' }),
+        icon: Repeat,
+        onClick: () => {
+          toggleActionSidebarOff();
```

Comment:
Hi! Thank you for this PR 🥰 . 

Thanks for moving items outside of JSX! Just want to propose moving all of this logic to the useGetItems custom hook, it would help to clean up components a little bit

Reply:
Hi @Nortsova ❤️  sure, that's a great idea! On it 🫡